File: sandbox/ruby/src/FaxLineListDefaultExample.rb
require "dropbox-sign"

Dropbox::Sign.configure do |config|
end

begin
    response = Dropbox::Sign::FaxLineApi.new.fax_line_list(
        {
            account_id: "ab55cd14a97219e36b5ff5fe23f2f9329b0c1e97",
            page: 1,
            page_size: 20,
            show_team_lines: nil,
        },
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling FaxLine#fax_line_list: #{e}"
end

File: sandbox/node/src/FaxLineListDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.FaxLineApi();

const accountId = "ab55cd14a97219e36b5ff5fe23f2f9329b0c1e97";
const page = 1;
const pageSize = 20;
const showTeamLines = undefined;

apiCaller.faxLineList(
    accountId,
    page,
    pageSize,
    showTeamLines,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling FaxLine#faxLineList:");
  console.log(error.body);
});
